packages/Cmfcmf-React-Core.package/CMFReactNodeComponent.class/instance/type.st

```diff
@@ -1,5 +1,8 @@
 accessing
 type
 
-	self flag: #todo. "I think using blockClosure hash here is not quite correct, since we technically do not care about the contents of the BlockClosure."
-	^ {#component. elementClass. selector. blockClosure ifNotNil: #hash}
\ No newline at end of file
+	^ {
+		#component.
+		elementClass.
+		selector.
+		blockClosure ifNotNil: [blockClosure startpc hash bitXor: blockClosure homeMethod selector hash]}
```

Comment:
Should this also include the class name of the `homeMethod` and not just the selector?

Reply:
Good point. I updated it so that we don't have all the nil values every time. Meaning the difference is now if we have four values, it is a block, otherwise a method.

Reply:
Thanks :)